_posts/2020-11-05-welcome-to-jekyll-demo-post.markdown

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title:  "Bioinformatic files: types and formats"
+date:   2020-11-06 12:54:04 -0700
+categories: blogpost
+---
+# Introduction
+In bioinformatics there are many different file types to store information from sequencing and -omics work. The most common file types are generally FASTA and FASTQ. But there are a lot of other semi-common file types as well like SAM, BAM, VCF as well as countless other self made file types for individual purposes.
+
+In this post I will cover four file types that are covered in the BTEC 640 course. The first two are the most common which are FASTA and FASTQ. The second two are slightly less common but still worked with frequently which are SAM and VCF.
+
+# FASTA
+
+FASTA stands for FAST-All. It was built off an earlier FASTP format that was designed around protein sequences and then expanded to encompass any kind of sequence format. Hence why the descriptor "All".  
+```fastq
+>XR_002086427.1 Candida albicans SC5314 uncharacterized ncRNA (SCR1), ncRNA
+
+TGGCTGTGATGGCTTTTAGCGGAAGCGCGCTGTTCGCGTACCTGCTGTTTGTTGAAAATTTAAGAGCAAAGTGTCCGGCTCGATCCCTGCGAATTGAATTCTGAACGCTAGAGTAATCAGTGTCTTTCAAGTTCTGGTAATGTTTAGCATAACCACTGGAGGGAAGCAATTCAGCACAGTAATGCTAATCGTGGTGGAGGCGAATCCGGATGGCACCTTGTTTGTTGATAAATAGTGCGGTATCTAGTGTTGCAACTCTATTTTT
+```
+
+The main format of a FASTA file is first a **comment line** that starts with '>'.
+You can put anything in the comment line as you see fit although typically there is there is species and index information.
+
+The second part is the **sequence line** which is a series of letters that represents the sequence of either a DNA strand, RNA strand, peptide strand, or something similar. Any line that starts after a semicolon ';' will be ignored. Also any of the following symbols in the sequence will also be ignored during processing: '(tab)','(space)','(asterisk)'.
+
+# FASTQ
+A FASTQ is similar to a FASTA except it also has indicators for quality scores. The quality scores indicate how reliability the read is for that particular part of the sequence. The 'Q' in this case stands for quality
+```fastq
```

Comment:
I don't think Jekyll understands sequence formats as a chunk type. I would delete `fastq` after the backticks -- it only understands coding languages for syntax highlighting.